rts/lra/coordinator/pom.xml

```diff
@@ -18,13 +18,9 @@
             -Djboss.bind.address=[::1] -Djboss.bind.address.management=[::1] -Djboss.bind.address.unsecure=[::1]
             -Djava.net.preferIPv4Stack=false -Djava.net.preferIPv6Addresses=true
         </ipv6.server.jvm.args>
-        <modular.jdk.args/>
-        <modular.jdk.props/>
-        <!-- the resource path of the coordinator - must match the deployment URL (see arquillian.xml) -->
+        <lra.coordinator.host>localhost</lra.coordinator.host>
+        <lra.coordinator.port>8080</lra.coordinator.port>
```

Comment:
is this still used since you've switched to URL?

Reply:
you are right, the `port` won't be needed, that's overridden with use of `lra.coordinator.url`